docs/api/python/spark_api.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: docu
+title: Spark API
+---
+
+DuckDB offers a PySpark-compatible API.
+
+> This extension is currently in an experimental state. Feel free to try it out, but be aware some things may not work as expected.
+
+## Usage
+
+```python
+import duckdb.experimental.spark
```

Comment:
Can we add the example from the release post here? And maybe link to the appropriate pyspark docs?